internal/common/db.go

```diff
@@ -65,26 +80,41 @@ func GetClusterFromDB(db *gorm.DB, clusterId strfmt.UUID, eagerLoading EagerLoad
 }
 
 func GetClusterFromDBWithoutDisabledHosts(db *gorm.DB, clusterId strfmt.UUID) (*Cluster, error) {
-	db = LoadHostsFromDB(db, "status <> ?", models.HostStatusDisabled)
+	db = LoadTableFromDB(db, HostsTable, "status <> ?", models.HostStatusDisabled)
 	return GetClusterFromDB(db, clusterId, SkipEagerLoading)
 }
 
-func GetClusterFromDBWhere(db *gorm.DB, eagerLoading EagerLoadingState, includeDeleted DeleteRecordsState, where ...interface{}) (*Cluster, error) {
-	var cluster Cluster
-
+func handleDB(db *gorm.DB, eagerLoading EagerLoadingState, includeDeleted DeleteRecordsState) *gorm.DB {
```

Comment:
Can this have a better name?
`handleDB` is rather meaningless and this seems to be doing something very cluster-specific.

Reply:
How about `prepareClusterDB`?